tests/profile.spec.js

```diff
@@ -155,3 +155,21 @@ test.describe("accessibility tests (dark)", () => {
     expect(accessibilityScanResults.violations).toEqual([]);
   });
 });
+
+test("Link clicks increase", async ({page}) => {
+  await connectMongo();
+  await page.goto("/eddiejaoude");
+  
+  const startingLink = await Link.findOne({  url: "https://twitter.com/eddiejaoude" })
```

Comment:
I would recommend using the username also here to avoid any confusion in the test data potentially later

Reply:
`test("Link clicks increase", async ({page}) => {

  await connectMongo();
  await page.goto("/eddiejaoude");
  
  const startingLinks = await Link.find({ username: "eddiejaoude" })
  const startingLink = startingLinks[0]

  const previousClickCount = startingLink.clicks; 

  const profileLink = page.locator('a').filter({ hasText: startingLink.name })

  await profileLink.click();
  await page.waitForTimeout(1000);
  const currentLink = await Link.findOne({ name: startingLink.name, username: "eddiejaoude" })
  const updateCurrentLinkClicks = currentLink.clicks
  
  expect(updateCurrentLinkClicks).toEqual((previousClickCount + 1));
});`

Reply:
This version is more dynamic/flexible.

Reply:
This looks great 👍